### DISCUSSION TITLE
Is there a list of scripts for voice files.

### DISCUSSION THREAD

**Discussion by vincemic** [gpstar81/GPStar-proton-pack#250] [Q&A]
I would like to regenerate the WAV files for the voice prompts using a different voice generator tool like https://ttsautomate.com .  Is there an enumeration of the voice prompts and scripts? No big deal if there isn't, it would be easy to create.

**Comment by gpstar81** [comment]
Kinda of,

https://github.com/gpstar81/GPStar-proton-pack/tree/main/sounds

All the voice prompts start with ###_VOICE_ and usually have a brief description of what it is saying.

There is no written script, but grab all the ###_VOICE_ files and just copy it when generating new ones.

There are 2 exceptions, the 1 to 60 counting is just _1, then _2 etc. It starts at #181, and level_1, level_2, level_3, level_4 and level_5 oh and also 243_eeprom_config_menu and 244_eeprom_led_menu

**Comment by nomakewan** [comment]
Here is a pastebin with all of the voice prompt lines as of firmware v5.3.3: https://pastebin.com/2LqXxPqq

Hope it helps!
